src/components/home/Hero/style.module.scss

```diff
@@ -0,0 +1,154 @@
+@use 'src/styles/vars.scss' as vars;
+
+.inline {
+  display: inline;
+}
+
+.hero {
+  min-height: min(vars.$min-content-height, calc(1080px - 6.25rem));
+  /* Header height: 4.25 rem, padding: 2 rem */
+  position: relative;
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: 5rem;
+    height: 100%;
+    padding: 0 1rem;
+    position: relative;
+    z-index: 1;
+
+    .header {
+      display: flex;
+      flex-direction: column;
+      gap: 0.25rem;
+
+      .headline {
+        display: inline;
+      }
+    }
+
+    .actions {
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      max-width: fit-content;
+      padding-bottom: 2rem;
+
+      .checkin {
+        background-color: var(--theme-accent-line-2-transparent);
+        border-radius: 1rem;
+        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
+        display: flex;
+        flex-direction: column;
+        gap: 0.75rem;
+        padding: 2rem;
+        transition: box-shadow 0.2s;
+
+        &:hover {
+          box-shadow: 0 5px 10px rgba(0, 0, 0, 0.3);
+        }
```

Comment:
it's a bit weird having a hover effect for the check-in card since clicking it doesn't do anything